src/data-collector/assets/config.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
why do we have two different config files? the original one is on src/assets

Reply:
I splitted it into multiple config files in different packages, Just to keep thing separate and clean. 
Maybe we could have one central config file and delete all this.